test/unittests/model/version/test_version_range.py

```diff
@@ -77,9 +83,13 @@ def test_range(version_range, conditions, versions_in, versions_out):
     ['*', False, ["1.5.1"], ["1.5.1-pre1", "2.1-pre1"]],
     ['*', None, ["1.5.1"], ["1.5.1-pre1", "2.1-pre1"]],
 
-    ['*, include_prerelease', True, ["1.5.1", "1.5.1-pre1", "2.1-pre1"], []],
-    ['*, include_prerelease', False, ["1.5.1"], ["1.5.1-pre1", "2.1-pre1"]],
-    ['*, include_prerelease', None, ["1.5.1", "1.5.1-pre1", "2.1-pre1"], []],
+    ['*, include_prerelease', True, ["1.5.1", "1.5.1-pre1", "2.1-pre1", "*"], []],
+    ['*, include_prerelease', False, ["1.5.1", "*"], ["1.5.1-pre1", "2.1-pre1"]],
```

Comment:
It seems you are adding ``*`` as a version to check, but this is really an invalid version, so this shouldn't be a case at all.

Reply:
I'm happy to remove it again, but just out of curiosity, why would `*` be an invalid version to check here?
My rational behind adding it was, to check that an "any" range is still contained / valid when compared to another "any" version range.

Reply:
Because if you try to create a package with ``version = "*"`` it will fail with:
```bash
ERROR: Invalid package version '*'
```
So the version ``"*"`` cannot exist at all. The values in ``["1.5.1", "*"]`` are exact versions, not version ranges, to validate if they are inside the range.

Reply:
Ah, makes sense! I'll remove it! Thanks for the clarification!